### ISSUE TITLE
Extracts from Kenya 

### ISSUE THREAD

**Issue by Kjmondo** [OpenSourceMycetoma/ARCHIVED_What-other-molecules-to-screen#9]
We are a team of scientists based in Mount Kenya University and Jomo Kenyatta University of Agriculture and Technology – Kenya. We are specialized in Phytochemistry for the preparation of extracts and the Pharmaceutical Screening for the search for bioactive molecules in order to isolate substances that may be of therapeutic interest.
We are currently working on the following projects:

1.	Repurposing a food supplement: Phytexponent as antibacterial.
Phytexponent is a food supplement made from five medicinal plants, namely: Allium sativum (garlic), Triticum repens (couch grass), Echinacea purpurea (purple coneflower), Viola tricolor (wild pansy) and Matricaria chamomilla (chamomile). It is registered and marketed worldwide as immunomodulator. 
Some phytochemicals from some of these plants are well documented to exhibit antibacterial activity like  allicin, ajoenes, and allyl sulfides in  Allium sativum, Cyclotides in Viola tricolor. Our hypothesis is that these phytochemicals can exhibit very potent antibacterial activities, when working synergistically. Hence, we have successively tested Phytexponent against the following bacteria: Streptococcus pneumoniae ([clinical isolates]()),  Acinetobacter baumani multidrug resistant (clinical isolates), methicillin resistant resistant Staphylococcus aureus (ATCC12393) , Pseudomonas Aeruginosa (ATCC 27857), Staphylococcus aureus (ATCC 25923), Klebsiella pneumoniae (ATCC 27232), Proteus mirabilis (ATCC25232), Escherichia coli (ATCC 25922)
Encouraged by this broad antibacterial activity including against many multidrug resistant bacteria of this formulation, We would like to continue screening it against mycetoma causing agent.

2.	Gervason extracts
"CD and PT are aqueous plant extracts with significant antimicrobial efficacy. The extracts were obtained by hot maceration followed by lyophilisation in vacuo using a standard procedure. They can be dissolved in normal saline (0.9% sodium chloride) or dimethylsulphoxide (DMSO) (up to 1.4 %). Preliminary investigation reveal the two extracts have remarkable antibacterial activity against E. coli (both standard and clinical isolates), S. aureus (both standard and clinical isolates), P. aeruginosa (standard), S. enteriditis (both standard and clinical isolates), K. pneumoniae (both standard and clinical isolates), B. subtilis (both standard and clinical isolates), among others. They also exhibit considerable efficacy against S. cerevisiae and C. albicans. Besides, CD posses considerable antidiarrheal and anti-inflammatory efficacy in animal model. PT has remarkable antioxidant, cognitive-enhancing, analgesic, and anti-inflammatory efficacy. Both extract do not elicit dermal or acute oral toxicity, and are safe to experimental animals (LD50>2000 mg/Kg BW). Phytochemical screening showed the present of various bioactive amalgams, such as flavonoids, tannins, alkaloids, coumarins, phenols, among others. We valorize these extracts as potential sources of efficacious lead compounds for developing drugs for treating various diseases."

3. Julia's extracts
XA-Ximenia americana, SA-Sodom apple (Calotropsis procera) and CM-Croton megalocarpus are methanolic plant extracts that we had previously tested their antioxidant activity. 
The extracts were obtained by cold maceration and followed by lyophilization in vacuo using standard procedure. They can be weighed and dissolved in methanol and DMSO4 to make a workable solution. I have not done any antimicrobial assays with all three in our laboratory. 
XA was picked from Loitokitok, southern Kenya, SA was collected from Kirinyaga, central Kenya and CM from Juja central Kenya. These plants grow well in tropical countries and have been used in traditional medicine to manage respiratory infections, toothaches, clean wounds, relieve stomach aches, and many other indications which point to antimicrobial activity of these extracts. We hope that this antimicrobial activity may point to compounds that will target the eumycetoma-causing fungal species.

**Comment by mattodd** [comment]
That's great @Kjmondo! I think you have already sent samples to @wwjvdsande, is that right? If so, can you please provide some details of what these are? We need to add them to our master list of samples.

**Comment by Kjmondo** [comment]
We will be sending 8 samples. All of them are plant extracts.  Prof. Wendy will decide on the concentration for the samples that have not been assayed for antimicrobial activity (We had discussed this on our earlier Zoom meeting). 
The samples are as below:
1. Phytexponent
2. CD (Cucumis dipsaceus) aqueous extract
3. PT (Piliostigma thonningii) aqueous extract
4. CD (Cucumis dipsaceus) alcoholic extract
5. PT (Piliostigma thonningii) alcoholic extract
6. XA-Ximenia americana methanol extract
7. SA-Sodom apple (Calotropsis procera) methanol extract
8. CM-Croton megalocarpus methanol extract

Will this help?

**Comment by mattodd** [comment]
OK, thanks for the details @Kjmondo. 

Calling medchem/informatics expertise @drc007 @loriferrins @cdsouthan: What do we do about numbering/labelling/listing extracts rather than pure samples? i.e. how does this dovetail with the Master List and our numbering system? Do we assign a MyOS code to the sample (with a batch number) and just omit the SMILES etc?

(also confirming I've added @epatwahirwa (Epaphrodite TWAHIRWA, Mount Kenya University) to this project so that he can also contribute).

**Comment by lferrins** [comment]
I think your plan to assign a MyOS code and omit the SMILES is a reasonable strategy - we do this routinely when screening blind samples for example. I've been trying to think if there would be a way that you could associate additional MyOS numbers should you go ahead and separate the extract but I haven't yet come up with a great way to do that (unless your numbering system might allow XXX-1 etc?).

**Comment by drc007** [comment]
Using the MyOS number as the sample code is reasonable, I've not looked at the master spreadsheet but does it have a "source" column? If so this could be used to identify the original source and then if you fractionate you could assign a new MyOS number to the extract and put the original MyOS number in the "source" column. Only when you get to an pure isolated and characterised compound would you need to include SMILES.

**Comment by MFernflower** [comment]
@bebi78  You have any experience with these herbs?

**Comment by bebi78** [comment]
@MFernflower No, but it is an interesting collection. OK, garlic is a common spice and applied in Ayurveda for the treament of heart diseases. Indian scientists reported that garlic paste can be used for the treatment of oral candidiasis. Might add well to the active DIM compounds, because DIM is formed in the stomach upon broccoli consumption from a indoleglycoside precursor of broccoli plants.

**Comment by MFernflower** [comment]
@bebi78 I did a little search on Cucumis dipsaceus - It's '
















'
'""""

**Comment by MFernflower** [comment]
@bebi78 I did a little search on Cucumis dipsaceus - It's got a bunch of steroid surfactant like compounds in it  - the fruits actively foam in water!  - probably too toxic for in-the-body use but might be something to consider looking into for a topical

**Comment by epatwahirwa** [comment]
I can give more information about the phytexponent formulation. It contains the following plants: Allium sativum (garlic), Triticum repens (couch grass), Echinacea purpurea (purple coneflower), Viola tricolor (wild pansy) and Matricaria chamomilla (chamomile).  All these plants are well documented in European Pharmacopoeia. In vitro, in our laboratory, the formulation has shown the very strong antibacterial and antifungal activities

**Comment by epatwahirwa** [comment]
Our hypothesis:  allicin, ajoenes, and allyl sulfides from Allium sativum, Cyclotides from Viola tricolor are the molecules responsible for these activities.

**Comment by MFernflower** [comment]
With a complex mixture like a plant extract there isn't a known MW - Might
be wise to screen as v/v or w/v

**Comment by bebi78** [comment]
@MFernflower @epatwahirwa Yes, these active ingredients allicin and allyl sulfides also have cancer preventive effects. So might be interesting to see their effects on mycetoma. I agree that such extracts or compound mixtures should be investigated by using µg/ml units.

**Comment by epatwahirwa** [comment]
@beni78, we have observed through patients taking this formulation that it has indeed effect on cancer. If you look to cyclotides, you will find that they have also anticancer activity, but as you say let's focus on mycetoma

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by bebi78** [comment]
@Kjmondo @epatwahirwa Are you in contact with Dr. Joshua Ikoni Ogaji of the Dept. of Pharmacy & Complementary/Alternative Medicine of the Kenyatta University in Nairobi?

**Comment by wwjvdsande** [comment]
> With a complex mixture like a plant extract there isn't a known MW - Might be wise to screen as v/v or w/v @MA-Jjingyi

You are correct. We will screen v/v in case of fluids and w/v in case of solids.

**Comment by MA-Jjingyi** [comment]
> > With a complex mixture like a plant extract there isn't a known MW - Might be wise to screen as v/v or w/v @MA-Jjingyi
> 
> You are correct. We will screen v/v in case of fluids and w/v in case of solids.

Yep! I will arrange them later.

**Comment by Kjmondo** [comment]
>@bebi78 I am not in contact with Dr. Joshua Ikoni Ogaji but I can look for him at Kenyatta university. Is he also working on antimicrobial plant mixtures?

**Comment by gmoriasi** [comment]
> @bebi78 I did a little search on Cucumis dipsaceus - It's got a bunch of steroid surfactant like compounds in it - the fruits actively foam in water! - probably too toxic for in-the-body use but might be something to consider looking into for a topical

Please note that these extracts do not cause Acute dermal/Oral Toxicity effects in animal model according to our recent study. Please find one of our recent publications attached herein.
[jhp-41251.pdf](https://github.com/OpenSourceMycetoma/What-other-molecules-to-screen/files/8394756/jhp-41251.pdf)

**Comment by bebi78** [comment]
@Kjmondo Thanks. Yes he also works on plant extracts. But I have seen now that he has left Kenyatta University and has moved to the University of Jos in Nigeria a few years ago.

**Comment by MFernflower** [comment]
> > @bebi78 I did a little search on Cucumis dipsaceus - It's got a bunch of steroid surfactant like compounds in it - the fruits actively foam in water! - probably too toxic for in-the-body use but might be something to consider looking into for a topical
> 
> Please note that these extracts do not cause Acute dermal/Oral Toxicity effects in animal model according to our recent study. Please find one of our recent publications attached herein. [jhp-41251.pdf](https://github.com/OpenSourceMycetoma/What-other-molecules-to-screen/files/8394756/jhp-41251.pdf)

It is pretty interesting to see that the cucumber isn't too toxic - then again there is a similar drug in the Americas called QS-21 https://en.wikipedia.org/wiki/QS-21

@bebi78 @Kjmondo @gmoriasi

**Comment by epatwahirwa** [comment]
@mattodd  and Wendy, any update on this extract from Kenya?

**Comment by MA-Jjingyi** [comment]
@epatwahirwa we did test the compounds at 100ug/ml and 25ug/ml. as for 2 liquid compounds, we used v/v%.  at these concentrations fungi growth were not inhibited. 

![11](https://user-images.githubusercontent.com/78109763/170984352-316b35ea-61b5-4d2c-afa1-9a0fcb6dfc22.png)

**Comment by gmoriasi** [comment]
Thank you @MA-Jjingyi for sharing this. Can I know what these values represent?

> @epatwahirwa we did test the compounds at 100ug/ml and 25ug/ml. as for 2 liquid compounds, we used v/v%. at these concentrations fungi growth were not inhibited.
> 
> ![11](https://user-images.githubusercontent.com/78109763/170984352-316b35ea-61b5-4d2c-afa1-9a0fcb6dfc22.png)

**Comment by MA-Jjingyi** [comment]
> @MA-Jjingyi Hello, thank you for your prompt response on the above asked questions. we have begun to understand the provided data, but from the response we can see that the higher concentration (100mcg/ml) has lower activity than the lower concentration (25 mcg/ml) for the PTM sample. can you elaborate about this outcome. thank you in advance in advance.

Hi @epatwahirwa . if i understand correct, the test be performed with positive control (only fungi growth, without any compounds added) and negative control (just medium)every time. the growth calculation also depends on the controls. in this case, the growth percentage is the relative values for growth, not the absolute values. and as usual, we need perform the same test 3 times(or 2 times). then, we use the mean value to do further decision.  @wwjvdsande am i correct?

**Comment by gmoriasi** [comment]
Hi @MA-Jjingyi , Thanks for your response. I am trying to understand these  results and not quite sure of some issues.
1. According to your protocol, you mention that the values presented indicate percentage growth of the microbe. What do the values that are >100?? does it mean the test drugs actually introduced some more colonies of the microbe? what does this actually mean?
2. If I got it right, does it mean that our test drugs were more efficacious at a lower concentration than at a higher concentration? ordinarily, the reverse is true.
3. Is it possible to perform the test in three replicates, and at at least three concentrations (using a serial dilution technique from 200 µg/ml for the powders and 10 % v/v for the Phytexponent) to better discern their efficacy and eliminate any experimental errors?
4. According to your protocol, why are you reporting the percentage growth rather than the percentage inhibition of growth upon treatment? Kindly clarify.
Regards

**Comment by wwjvdsande** [comment]
@gmoriasi @epatwahirwa 
1. Theoretically growth values >100% would mean that a compound enhances growth. However since this is a biological assay I would only really speak of enhancement if in all three replicates the value is above 150% or 200%. 
2. We test hyphae and there is some growth variation per assay and per well. Therefore we only use the cut-off value below 20% to say growht inhibition. The values between 20% and 80% often vary per replicate. Therefore we don't pay to much attention to these values. It simply means that some growth is inhibited but not all. If at some point it is higher or lower we don't look to much on it. With most compounds you indeed expect that the higher concentration you test the more efficient the compound is in inhibiting the fungal growth. However there are excemptions to this. For the echinocandins it is for instance known that there can be a paradoxal effect. Meaning that at lower concentrations there can be a higher growth reduction than at somewhat higher concentrations. This has to do with the stress response of the fungus which could alter the metabolism a bit and make a drug less effective. With your phytocompounds we look at a mixture of compounds so that makes it even more difficult. However, looking at these values I would not put to much emphasis on the actual values. What it shows is that at both concentrations tested there was no complete inhibition of fungal growth. 
3. The highest concentration we tested was the highest we could do for now. The compounds were difficult to dissolve and we cannot add to much DMSO to the assay. DMSO itself will kill the fungus so that means that we will demonstrate killing but due to the solvent used.
4. We are reporting percentage growth. If you want percentage inhibition it is simply the following formula: -value reported = percentage inhibition. So in case of PTM that would be 100-89=11% inhibition.

**Comment by Kjmondo** [comment]
@bebi78 I will inform my colleagues, Thank you.
